config/locales/uk.yml

```diff
@@ -0,0 +1,49 @@
+uk:
+  errors:
+    messages:
+      taken_in_past: 'раніше використовувався.'
+      equal_to_current_password: 'має відрізнятися від поточного паролю.'
+      password_complexity:
+        digit:
+          one: 'повинен включати хоча б одну цифру'
+          few: 'повинен включати хоча б %{count} цифри'
+          many: 'повинен включати хоча б %{count} цифр'
+          other: 'повинен включати хоча б %{count} цифри'
+        lower:
+          one: 'повинен включати хоча б одну малу букву'
+          few: 'повинен включати хоча б %{count} малі букви'
+          many: 'повинен включати хоча б %{count} малих букв'
+          other: 'повинен включати хоча б %{count} малі букви'
+        symbol:
+          one: 'повинен включати хоча б один розділовий знак або символ'
+          few: 'повинен включати хоча б %{count} розділові знаки або символи'
+          many: 'повинен включати хоча б %{count} розділових знаків або символів'
+          other: 'повинен включати хоча б %{count} розділові знаки або символи'
+        upper:
+          one: 'повинен включати хоча б одну прописну букву'
+          few: 'повинен включати хоча б %{count} прописні букви'
+          many: 'повинен включати хоча б %{count} прописних букв'
+          other: 'повинен включати хоча б %{count} прописні букви'
+  devise:
+    invalid_captcha: 'Неправильне введення капчі.'
```

Comment:
I'm pretty sure this is Google Translate not working well, but it's saying 'Incorrect introduction of captcha." Is it fair to assume this would be understood as something like "input?"

Reply:
changed a little to be more understandable